zombienet/tests/0007-full_node_warp_sync.feature

```diff
@@ -0,0 +1,14 @@
+Description: Full node catching up using warp sync
+Network: ./0007-full_node_warp_sync.toml
+Creds: config
+
+alice: is up within 30 seconds
+bob: is up within 30 seconds
+charlie: is up within 30 seconds
+dave: is up within 30 seconds
+eve: is up within 30 seconds
+ferdie: is up within 30 seconds
+alice: parachain 2000 is registered within 225 seconds
+one: reports block height is at least 7 within 225 seconds
```

Comment:
These values are too low, please correct them so that we are actually testing that we warp synced close to the tip. Maybe 1000 blocks or so.

Reply:
The snapshot I provided only goes to around 760 blocks. I hope that is acceptable, if I can generate a bigger one